core/src/main/java/org/bitcoinj/core/Utils.java

```diff
@@ -62,17 +62,34 @@
      * The regular {@link java.math.BigInteger#toByteArray()} method isn't quite what we often need: it appends a
      * leading zero to indicate that the number is positive and may need padding.
      *
+     * <p>
```

Comment:
Thanks for this JavaDoc explanation, it's much appreciated. <p> needs to be closed though.

Maybe we can merge the old first paragraph with your second? It seems to explain the same thing, but yours is more understandable.